env0/data_configuration_variable.go

```diff
@@ -77,6 +77,15 @@ func dataConfigurationVariable() *schema.Resource {
 				Description: "scope of the variable",
 				Computed:    true,
 			},
+			"enum": {
+				Type:        schema.TypeList,
+				Description: "limit possible values to values from this list",
```

Comment:
Since it's the `data`, it's just for reading

```suggestion
				Description: "possible values of this variable",
```

Reply:
ac80073f